GameOfLife/Calculations/CellPopulator.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace GameOfLife
+{
+
+    public class CellPopulator
+    {
+        /// <summary>
+        /// Populates array with random numbers either 1 or 0. Live cell = 1. Dead
+        /// cell = 0.
+        /// </summary>
+        /// <param name="gameField"> Starting array gameField created 
+        /// based on user entered lenght and width</param>
+        /// <returns>gamefield array with randomly populated cells (1 or 0)</returns>
+        public int[,] RandomizeCells(int[,] gameField)
+        {
+            Random randomizer = new Random();
+
+            for (int i = 0; i < gameField.GetLength(0); i++)
+            {
+                for (int j = 0; j < gameField.GetLength(1); j++)
+                {
+                   gameField[i, j] = randomizer.Next(0, 2); 
+                  // GliderCreation(gameField);
+
+                }
+
+            }
+            return gameField;
```

Comment:
Add empty line because it is after code block.

Reply:
Added emty line before return